### PR TITLE
Add lookup_table op and its grad to ngraph engine

### PR DESCRIPTION
**Pull request by baojun-nervana**
Added lookup_table and lookup_grad op. The ngraph is updated to support those ops. The ops are needed for BERT model

### CODE REVIEW

**Review by mozga-intel — APPROVED** [review]
LGTM!

File: cmake/external/ngraph.cmake [hunk 1]
```diff
@@ -37,7 +37,7 @@ INCLUDE(GNUInstallDirs)
 INCLUDE(ExternalProject)
 
 SET(NGRAPH_PROJECT         "extern_ngraph")
-SET(NGRAPH_GIT_TAG         "096ad6ef0c04d57db1522940dbdf9a0652768065")
+SET(NGRAPH_GIT_TAG         "4ec94acc11084a5d53418f565529310fa584899a")
```
Comment (tensor-tang) [inline cmake/external/ngraph.cmake:40]: why change this？

Comment (baojun-nervana) [reply cmake/external/ngraph.cmake:40]: The ngraph::op::ScatterAdd (used in this lookup_table_op) is supported with new ngraph version.

**Review by tensor-tang — APPROVED** [review]
LGTM

**Pull request merged**